frontend/src/app/(logged-in)/profile/_components/GoalsCard.tsx

```diff
@@ -42,24 +59,17 @@ export default function GoalsCard() {
                         <div>
                             <div className='mb-2 flex items-center justify-between '>
                                 <span className='text-md font-medium'>Total Distance</span>
-                                <span className='text-sm text-muted-foreground'>56.7/80km</span>
+                                <span className='text-sm text-muted-foreground'>56.7/{goalOfEachMonth ? goalOfEachMonth.total_distance : ' -- '}km</span>
                             </div>
                             <Progress value={70} className='h-2' />
                         </div>
                         <div>
                             <div className='mb-2 flex items-center justify-between '>
                                 <span className='text-md font-medium'>Average Pace</span>
-                                <span className='text-sm text-muted-foreground'>5'30"/km</span>
+                                <span className='text-sm text-muted-foreground'>5'30"/{goalOfEachMonth ? goalOfEachMonth.average_pace : ' -- '}km</span>
                             </div>
```

Comment:
@Higesennine I added test data and checked, but it seems that the average pace isn't showing. Also, I have a question: currently, the profile page isn't showing my profile. Is someone supposed to fix it later?
![854ae25aaf01e4a805a7b2036375bb63](https://github.com/user-attachments/assets/1df03b11-34ff-4207-b371-a1e88248feab)
<img width="1400" alt="Screenshot 2024-12-09 at 20 30 16" src="https://github.com/user-attachments/assets/89c2b2ee-f76b-42b6-b9df-bd6c1d1212c6">